src/control/common/proto/mgmt/acl.pb.go

```diff
@@ -0,0 +1,122 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: acl.proto
+
+package mgmt
+
+import proto "github.com/golang/protobuf/proto"
+import fmt "fmt"
+import math "math"
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
+
+type GetACLReq struct {
+	Uuid                 string   `protobuf:"bytes,1,opt,name=uuid,proto3" json:"uuid,omitempty"`
+	XXX_NoUnkeyedLiteral struct{} `json:"-"`
+	XXX_unrecognized     []byte   `json:"-"`
+	XXX_sizecache        int32    `json:"-"`
+}
+
+func (m *GetACLReq) Reset()         { *m = GetACLReq{} }
+func (m *GetACLReq) String() string { return proto.CompactTextString(m) }
+func (*GetACLReq) ProtoMessage()    {}
+func (*GetACLReq) Descriptor() ([]byte, []int) {
+	return fileDescriptor_acl_0d5d9c26e3f1e88c, []int{0}
+}
+func (m *GetACLReq) XXX_Unmarshal(b []byte) error {
+	return xxx_messageInfo_GetACLReq.Unmarshal(m, b)
+}
+func (m *GetACLReq) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	return xxx_messageInfo_GetACLReq.Marshal(b, m, deterministic)
+}
+func (dst *GetACLReq) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_GetACLReq.Merge(dst, src)
+}
+func (m *GetACLReq) XXX_Size() int {
+	return xxx_messageInfo_GetACLReq.Size(m)
+}
+func (m *GetACLReq) XXX_DiscardUnknown() {
+	xxx_messageInfo_GetACLReq.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_GetACLReq proto.InternalMessageInfo
+
+func (m *GetACLReq) GetUuid() string {
+	if m != nil {
+		return m.Uuid
+	}
+	return ""
+}
+
+type GetACLResp struct {
+	Status               int32    `protobuf:"varint,1,opt,name=status,proto3" json:"status,omitempty"`
```

Comment:
Why were the error codes removed? The old acl.pb had an error code for invalid uuid which I don't see an equivalent here.

Reply:
We're just returning the DAOS error codes, similar to what's done in the other mgmt requests. Since UUID is the only input, I figured -DER_INVAL was descriptive enough.